crates/aptos-crypto/src/unit_tests/bls12381_test.rs

```diff
@@ -347,3 +347,47 @@ fn bls12381_sample_signature_verifies() {
 
     assert!(sig.verify_arbitrary_msg(b"Hello Aptos!", &pk).is_ok());
 }
+
+#[test]
+fn bls12381_sample_aggregate_pk_and_multisig() {
```

Comment:
this is not a test but an example? if so you can leverage rust's doc test for that

Reply:
Yes, it's not a test: it's a way for me to generate sample signatures and public keys to test the Move implementation. I ended up putting this here as a test so I don't have to write a separate tool. I did something similar for Ed25519 too.

Should I create separate command-line apps/tools to generate these samples? See the other examples in signature.move

Reply:
cli is probably too overkill for now

Reply:
Probably. I added an #[ignore] to the tests so they do not slow us down in CI. They can still be run in cargo. For example:
```
cargo test -- bls12381_sample_aggregate_pk --nocapture --include-ignored
```